orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -1152,18 +1142,6 @@ private int initControlConnection (int maxTries, boolean isReconnect) throws Exc
                         // implementations for all the events we don't care about.
                         logNotice( "adding control port event handler");
 
-                        if (Prefs.useDebugLogging()) {
-                            conn.setDebugging(System.out);
-                            conn.addRawEventListener(new RawEventListener() {
-                                @Override
-                                public void onEvent(String keyword, String data) {
-
-
-                                    debug(keyword + ": " + data);
```

Comment:
@n8fr8  This line here seemed to log each log statement, I deleted it because logging information seemed to come from syslog later on without us needing this code This ultimately fixes the bug

Reply:
Yeah, I think I left that in accidentally, trying to fix the hang issue.